Identity_Presentation/Controllers/ClaimController.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Identity_Application.Commands.Claims;
+using Identity_Application.Models.BaseEntitiesModels;
+using Identity_Application.Queries.Claims;
+using Identity_Domain.Entities.Enums;
+using Identity_Infrastructure.Authentication;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Identity_Presentation.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
```

Comment:
route is api/Claim/something?

Reply:
api/claim/ 
     (Claim) (Case doesn`t matter)
api/claim/{id} - optionally for some endpoints